### DISCUSSION TITLE
Values to train a style.

### DISCUSSION THREAD

**Discussion by TetraOrb** [TheLastBen/fast-stable-diffusion#1030] [Q&A]
Hello! I tried to train a style, but the final result looks nothing like the style I want. So I'd like some tips on values. I use a set of 288 pictures. When I use all of them as instance pictures without I get a problem with faces, that do not stylize at less than 7000 steps.

So I tried to divide them into two parts - 50 instance images and 238 concept images. I used 10000 steps, 350 text encoder and 2000 concept encoder, but the result looks nothing like the style I wanted.

Here is the comparison of what I want and what SD draws after training. I'd be happy to get help to adjust the values to get the style closer to mine.)
![tetra (016)](https://user-images.githubusercontent.com/115459677/207789997-b1a4b8b0-a6c3-487a-a54d-437cf1fb587d.jpg)
![image](https://user-images.githubusercontent.com/115459677/207789911-3de741b8-5baf-4f6b-b942-7486efbb5fb6.png)

**Comment by TheLastBen** [comment]
10.000 steps is too much, reduce the instance images to 15-20 and the steps to 3000, keep the text_encoder to 350

**Reply by TetraOrb** [reply]
I see, thank you! May i ask, how many concept training steps to use?

**Reply by TheLastBen** [reply]
you can safely use 1000 steps per 200 images, increase the steps if you're not getting enough composition diversity in the output

**Reply by TetraOrb** [reply]
Tested those values today - nope, the result is still disappointing. The style looks nothing like mine... =( should I just train futher?

**Reply by TheLastBen** [reply]
send me the instance images I'll try to train a model

**Reply by TetraOrb** [reply]
Here they are: https://drive.google.com/drive/folders/1DzZ022a_fZqjcjsUjDSpGPUqg6JVBO4j?usp=share_link

**Reply by TetraOrb** [reply]
I was trying to train 1.5 version.

**Reply by TheLastBen** [reply]
the quality of the instance images is terrible, do you have the original uncropped ones ?

**Reply by TetraOrb** [reply]
Yes, of course. I added the Uncropped folders with originals.  But if the problem is "grain" it was always part of the style. And I believe when I was training on the same crops at the end of October -  beginning of November it was not an issue at all. The style resemblence was pretty close, not perfect, but not something completely different. Unfortunately I deleted my models, thinking I won't need them, and now I cannot train a new one with the same quality outputs. Hope we'll be able to find a solution! Thank you for your time and help.

**Reply by TheLastBen** [reply]
I don't see the uncropped images

**Reply by TetraOrb** [reply]
Here are the links: 
concept https://drive.google.com/drive/folders/12o-hwVfNHO8Xa3RtibaN7_1QpIlaldFB

and instance
https://drive.google.com/drive/folders/1Ot18TRsOlpCGjw4F6r--Oy70NjpwZlkn

**Reply by TheLastBen** [reply]
the black & white concept images are wrong for the training, black&white in itself is a concept

**Reply by TheLastBen** [reply]
I'll train a model with the instance images.

**Reply by TetraOrb** [reply]
Hm..i see, in the previous models i actually got black and white outputs with occasional splash of color, but it was ok)) i understand that the color affects the result this way)

**Reply by TheLastBen** [reply]
![tmp4qgak8yk](https://user-images.githubusercontent.com/110583491/208106143-267a33df-da51-4725-80e4-69aba5b102bc.png)
![tmpb79p2axt](https://user-images.githubusercontent.com/110583491/208106161-925e0bd6-8566-4b03-a21e-35731b74d39c.png)
![tmpjuew5_ip](https://user-images.githubusercontent.com/110583491/208106180-d8b33bc2-a33a-4ac8-ace0-d54c3e82c5f7.png)
![tmpyt6iuw43](https://user-images.githubusercontent.com/110583491/208106191-c64a12bb-8dea-48cf-ab01-2edc01a49c9e.png)

**Reply by TheLastBen** [reply]
here is the model https://drive.google.com/file/d/1-3utQxN5_R6uD5oDSmeiUmCqaCYrVL5u/view?usp=sharing but don't expect much from it as the instance images are inconsistent and low quality.
identifier : bbvvtt

**Reply by JJB1** [reply]
This could be an application for aesthetic gradients, works better for training an art style 

https://arxiv.org/abs/2209.12330

**Reply by nawnie** [reply]
i wanna try, can i get a link to all the original instance images?

**Reply by TheLastBen** [reply]
https://drive.google.com/drive/folders/1Ot18TRsOlpCGjw4F6r--Oy70NjpwZlkn
don't forget to rename them

**Reply by nawnie** [reply]
Oh no I was wanting his original set :P style is nice, but some of these pictures contain the same subject, I feel a little change up in source data and we can get a huge diffrence.

**Reply by TetraOrb** [reply]
> here is the model https://drive.google.com/file/d/1-3utQxN5_R6uD5oDSmeiUmCqaCYrVL5u/view?usp=sharing but don't expect much from it as the instance images are inconsistent and low quality.
> identifier : bbvvtt

Thank you! Let me ckeck it out!

**Reply by TetraOrb** [reply]
> Oh no I was wanting his original set :P style is nice, but some of these pictures contain the same subject, I feel a little change up in source data and we can get a huge diffrence.

Here are my pics: https://drive.google.com/drive/folders/1DzZ022a_fZqjcjsUjDSpGPUqg6JVBO4j?usp=share_link. Each folder has a folder with cropped and uncropped images inside)

**Reply by TetraOrb** [reply]
> This could be an application for aesthetic gradients, works better for training an art style
> 
> https://arxiv.org/abs/2209.12330

Sounds interesting! Thank you!

**Reply by nawnie** [reply]
i can try again with the full set if you want, if not the upscaled set i made from what you 
shared earlier is in my link underneath

**Comment by nawnie** [comment]
enjoy,
keyword is petrova, ive been using the prompt in xxx,the style of petrova 20 steps cfg 7 eurlerA
![01280-promptub_5jae3](https://user-images.githubusercontent.com/106923464/208800870-b109a9f3-16d1-42fe-aa6a-2dc292981745.png)
![01285-promptn_yxyuvh](https://user-images.githubusercontent.com/106923464/208800876-ced4529f-54c4-42aa-b556-d590952b5951.png)
![01287-promptyidx_0p7](https://user-images.githubusercontent.com/106923464/208800879-dea7ef08-1fee-41bf-822a-0bb75ea29c55.png)
![01289-prompt81e030_t](https://user-images.githubusercontent.com/106923464/208800880-9be38bd2-5db3-4c78-98c6-432f8822b4ef.png)

... imma ust say no adjustments prompt and go bud

https://drive.google.com/drive/folders/1GC81k3LI9RwHMcRLOt513MPEeIahGgU2?usp=sharing

**Reply by nawnie** [reply]
hmm looks blurrier big one sec
![01295-{prompt}](https://user-images.githubusercontent.com/106923464/208801414-f8b9c85f-c00f-4e9c-b05b-f6be3ae2c14f.png)

**Reply by nawnie** [reply]
if im being honest, it could use another round of training, i wanted to capitalize on style training, so my suggestion is to take images and run it at 700+ resolution for another 1000 steps saving every 200

**Reply by TheLastBen** [reply]
the instance images are of low quality so the model is mimicking that
